ballerina/tests/resources/compose.yaml

```diff
@@ -1,5 +1,3 @@
-version: '2'
```

Comment:
I think this version works with `docker compose` so we don't need to change it ?

Reply:
Yes, it works with the warning as a backward compatibility
The recommendation is not to use a version